plugins/modules/system/ssh_config.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2015, Björn Andersson
+# Copyright: (c) 2021, Ansible Project
+# Copyright: (c) 2021, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: ssh_config
+short_description: Manage SSH config for user
+description:
+    - Configures SSH hosts with special IdentityFiles and hostnames.
+author:
+    - Björn Andersson (@gaqzi)
+    - Abhijeet Kasurde (@Akasurde)
+options:
+  state:
+    description:
+      - Whether a host entry should exist or not.
+    default: present
+    choices: [ 'present', 'absent' ]
+    type: str
+  user:
+    description:
+      - Which user account this configuration file belongs to.
+      - If none given, C(/etc/ssh/ssh_config) is used.
+      - If a user is given, C(~/.ssh/config) is used.
```

Comment:
Shouldn't `user` and `ssh_config_file` be mutually exclusive (and reference each other in the docs)?